Agent.build now honors embedded_path for rtloader (#7616)
DataDog/datadog-agent: docs/dev/agent_build.md

to adjust the build for your specific configuration, add or edit the files within
the `pkg-config` folder.

## Testing Agent changes in containerized environments

> Building an Agent Docker image from strach through an embedded build is a slow process.

Reviewer: ```suggestion
Building an Agent Docker image from scratch through an embedded build is a slow process.
```

> Sometimes, you may want to quickly test a change or bufix in a containerized environment (Docker, Kubernetes, ECS, etc.).

Reviewer: ```suggestion
Sometimes, you may want to quickly test a change or bugfix in a containerized environment (Docker, Kubernetes, ECS, etc.).
```

> Sometimes, you may want to quickly test a change or bugfix in a containerized environment (Docker, Kubernetes, ECS, etc.).

Reviewer 2: ```suggestion
You can quickly test a change or bug fix in a containerized environment (such as Docker, Kubernetes, or ECS).
```

> One way to do it is to patch the Agent binary from official Docker image, with a simple Dockerfile:

Reviewer 2: ```suggestion
One way to do this is to patch the Agent binary from an official Docker image, with a Dockerfile:
```

```
FROM datadog/agent:<AGENT_VERSION>

COPY agent /opt/datadog-agent/bin/agent/agent
```

> However, for this to work properly, two things are important:

Reviewer 2: ```suggestion
For this to work properly, two things are important:
```

> - Your change needs to be done on top of `<AGENT_VERSION>` tag from our repository.

Reviewer 2: ```suggestion
- Your change needs to be done on top of the `<AGENT_VERSION>` tag from the DataDog repository.
```

> - You need to run invoke task with proper embedded path `inv -e agent.build -e /opt/datadog-agent/embedded`.

Reviewer 2: ```suggestion
- You need to run the invoke task with the proper embedded path `inv -e agent.build -e /opt/datadog-agent/embedded`.
```

> Note that this will make `invoke` to install the builds artifacts in `/opt/datadog-agent/embedded` folder. Make sure folder exists and current user has write permissions.

Reviewer 2: ```suggestion
**Note**: This makes `invoke` install the build's artifacts in the `/opt/datadog-agent/embedded` folder. Make sure the folder exists and the current user has write permissions.
```
